apis/react-suspense.md

```diff
@@ -25,13 +25,10 @@ that also supports key-based invalidation and preloading but they are not strict
 
 Caches should be idempotent and should **throw promises** to resolve data fetches.
 
-## maxDuration is NOT actual Duration
+~~## maxDuration is NOT actual Duration~~
```

Comment:
Would it be better if removed instead of strike-through?

Reply:
probably just remove it

Reply:
Thanks for the review. I removed it.